experiments/speedrun/llama_75m_z_loss/llama_75m_z_loss.py

```diff
@@ -1,6 +1,6 @@
 """
 Speedrun code for a 75M parameter model based on the LLaMA architecture.
-The run also experiments with setting z_loss_weight to 1e-4.
+The run experiments with setting z_loss_weight to 1e-4.
```

Comment:
```suggestion
We set z_loss_weight to 1e-4.
```
Maybe reference something or say another sentence about what this is for